backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5573480_sys_gh10279AddEnforceMarketingChannelsSysConfig.sql

```diff
@@ -0,0 +1,5 @@
+-- 2020-11-27T06:12:31.788Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541346,'O',TO_TIMESTAMP('2020-11-27 08:12:31','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.marketing.base','Y','de.metas.marketing.EnforceUserHasMarketingChannels',TO_TIMESTAMP('2020-11-27 08:12:31','YYYY-MM-DD HH24:MI:SS'),100,'N')
```

Comment:
please add a description that explains that this applies 
* only to not-system users
* only whn added in the user-window